pkg/kubelet/dockertools/docker.go

```diff
@@ -454,7 +454,7 @@ func GetDockerPodInfo(client DockerInterface, manifest api.PodSpec, podFullName
 	for _, container := range manifest.Containers {
 		expectedContainers[container.Name] = container
 	}
-	expectedContainers["net"] = api.Container{}
+	expectedContainers["POD"] = api.Container{}
```

Comment:
LGTM

On Mon, Jan 26, 2015 at 5:29 PM, Mrunal Patel <notifications@github.com>
wrote:

> In pkg/kubelet/dockertools/docker.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3817#discussion_r23580844>
> :
>
> > @@ -454,7 +454,7 @@ func GetDockerPodInfo(client DockerInterface, manifest api.PodSpec, podFullName
> >  	for _, container := range manifest.Containers {
> >  		expectedContainers[container.Name] = container
> >  	}
> > -	expectedContainers["net"] = api.Container{}
> > +	expectedContainers["POD"] = api.Container{}
>
> Introduced a constant with a comment saying it should match kubelet
> constant. Let me know if that works otherwise build is failing with import
> cycle.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3817/files#r23580844>
> .
>